notebooks/09-Importing-Data.ipynb

```diff
@@ -0,0 +1,1706 @@
JJ+eNH5Wb6qGZJLXeqCndk/27aMm0w\n7V2tCER6/KfeP42/zp1zjY0VIOaIP9f8K64zSpaiE6/uQ2XqRJ6X0rGao0EgvxHQhGZ8kzDJFTjC\nhCyuv4bQnN/Im/gNAgaBVEfAo2GCM6O//4YwkeolW7TkL7KECVnoU2Mp1WlH550HV4YwkVovgHcS\nziv788+PZbunXb03j+Lqvvvuo1EjR6qxIAbiesDHcVapUoU2btwYU+z2pImufxwKp8PYxEVeJq38\nEsXOqieffJIef/xxa5eV9PM9O0eQaN06dYXw4BeHvueW18lygEaMGG7bzIWKz1tuuUUHCTliEmz+\n/PlUunTpkGfmRuoiMGhqB34f1OSaqsWcF25KB3c0hInULdXYJV8ztLmUPwodtUA70xHWSJijQcAg\nYBCIjADaUfX9xDhEt6a5rPJ3jlybn/RGoF2vuZxB14DAOk+nhb6PP/qImmZnOwXpyu7ChQvpfNZU\nF805u/s5sDMYkWDTp0+ntm3bynkjNpuxmMnjbofFvNq1TxPTFdDMEM25NUyIXyvJBlFMcsDvmWfW\n43HVck8SSLJEiZK0f/9+XnOJnr4ncIIu7LEc4oMIqrnhsdwIeywXT1IhhAlX4LwSJtwmOVzRUTST\nHPBbtmxZgnaRYFexYkXepMCaNY7SoezUGNaqDBZ+iBabJbBpIpJLNP6R0lq+/EeqV6+u7QVVTmuO\naN26Nb3zzjv2M2iKxHxBsIP2EsxrlCpVKvhRUl7bpo2CpDMmOYIAScDl4T+30MbX+tDBLWs5Nu/4\nLzh6vCYZqH+i9gNXeH9UfeQDO3Uvo3Q5qtxpKB1z6ply1/wYBJIBASE0M7EHXQ6pwhCKK67ZGJIM\npWNkMAgYBJIZARDO/L7/pv1M5lIzsgUjUHQJE1M6UiCDdzBARxw669J5D7Aq+dAdJcGgmevkQUDs\n4vIknKj5Q0li9IWxF7tEapjAzpEzzjiDtm3bzjFbCUgq/MMzEff16kUPP/KIvhPxqCdN1ASGJTAf\n8rrLJ1JisKkKcxifffY5e4PcnJAkyQNcPoLpBxWtJ554YthotLzKgwSW0+BJth7de9CYsWNseMSn\n7T1At976Dxo3blzYdMyD1ENgELejwri33gm8f7msT3xIR2OSI/VKM36JFXNYEWakVbTKP2ugWeiL\nH00TwiBgECiKCKwZ1kL6YrKDTXXQBAZDPCsatQE7o2XXTdDYIp0W+hS5nXd6y5jAGX9gwBCLSQ7U\nhKeeeoruvPNOb6XggQzGf61bt6EZ78ygzz//nC6++GKVjGv8gXRGjhxFvXr19IYPc9WpUyeaMiWY\n+BtgkxwN6NtvI2uYaNq0KX3C2vuc8agWJMAL0L8LwT5Msvl6uxLMK26FOUgv/nnVbghNBZ1vvNF3\n/J13wkQDWrp0SQgOsRAmap9Wm1b9vErCCuIadq4NO3ZsF/MlIRHHceOnn36iOnXqeENY9a9jx440\naXLk+aNE4+8VJPSq8YWNaeGChfxAA8FH/l+ieHHasGEDgUgC0y6nVDuFNaJsYn/WHIHVDt3evTuN\nCWMGJzS1wr8j7SiPP0FIwreUN4Jz3QywSY7swhcujSQ4uOkX2jypPx36AySk0PGfgz/aZmScSwNs\nHZQLlwlsjgXPvxY/ripV6TyCSpxYDQGMMwgkDQJakybqsUOZyDWE5qQpISOIQcAgkKwIrLFMGgV/\n/2v0N/PEyVpmRq5QBIouYWJqR3toaI0NpSOf0yF4giQUNHMneRCwJ+F0IbpEG8saJromSMNEhw4d\nbLu36DLzcI/7zjwA5P8lWYXg6tW/0MmshjUWh0kKP9WdeZ20ipbmvn37CCo133//fenuy/jVFWjW\nrFnUooW2oe16YJ16CRNqyID8Dxs+lPr17WcHwO6bJhddRN9Yk4l6GK3wUt4SSWKxEzYnhYYANPVg\nop91FPK7YE2GsDRGU0+hFUmBJoyJBL/yNwt9BVoMJjGDgEEghRHADjYsJByRniV6TKqfZUxypHCh\nxiE6THKg/JURAl3+MMnBBIM0cRirNWNye/D4A9mLlTCxY8cOOoXHWfv2H7DHX3pEAnMca9asofvv\nv5+wkB88/ih1zDG0Yf16MZ0QC6TXM2FiMggTKA6X0NBeocc44eLp2bMnPf7Y41Ke7uA4/+TTT4XQ\nES5sft6vwoSJzT6aFvI69lQaJm5ikV0AWRnIM2Hi7LPFfIaNm3USC2HCLjOWIbj8Qbb597//fVTw\n5uTk0JDBgzm3LJSt4UQJOHr0aIKmhkgu0fhHSgvPnn32WbrjX/+yS8fGlJ89/fTTdMcdd9DcuXPF\nRI37GcLi+iMm/Vx66aW4TAnn0TAh41IuKa6abz+anRLyp4KQ+9csps1TB9PhA7vVK+Az/pd8WPhL\n5UNlYueuY2gxrNtU8qTaVOWG4VSs7Aniz/wYBJIJAU2YQJ1VWyyZAMQNS5YxPZpMxWRkMQgYBJIQ\nAd1+Bn//jUmjJCwsI1JYBIosYWIgNEzo3jqfgP3Mv7zQZ3ZGh60tSfjAPQlnN8bWSaIW5x1zExwx\nRt9w1gEjvttuvY1efPFFdT+G38qVKtEW7PLRTmZKcymvk1Y6mkjHzz77jC655BLxYuNkBXiEzYz0\nZg0Z4VzlSpVZXrWTwJ1/P3lXrVpF5553Lu3csdM3umN40hI7wM455xzf5+ZmaiEwaBq3o9x2ygSi\nFp0r2ODrDPFMw5HOR+yM5o8nZxGtiuX41Kha02CYo0HAIGAQiIwAVFZi/IF2FN9S2PcGES1rwKzI\nAc3TtECgba95rvLnWsDfUHxWZ6TRzmhFbs/mfDrDJyk8zuwCNtd3wQUXxFSWN97YmSZMmKj8cmQY\nkukeSI8ePWjc+JfoAAgVQe6mm24ijOViddokhyOvSqwhmxf89tvFEaOZNm0agWTv527s3Jlefe01\nv0f5
```

Comment:
If we use the parens here I think it should be backticked: `open()`

Reply:
A GH bug seems to be causing this comment to be shown in the wrong part of the file; it should be on line 1318